### ISSUE TITLE
Crash upon opening any book.

### ISSUE THREAD

**Issue by jamal-ali1989** [johnfactotum/foliate#1411]
**Describe the bug**
The app simply closes if choose to open any .epub or .djvu.
I am using the flatpak version on Linux Mint Cinnamon 22.

The error report is attached
[Foliate Error report.txt](https://github.com/user-attachments/files/17357471/Foliate.Error.report.txt)

**Comment by johnfactotum** [comment]
The GDB log doesn't seem to say anything? I don't see any paths or symbols. Anyway, if you're using NVIDIA it's most likely a WebKitGTK and/or NVIDIA bug. See https://github.com/johnfactotum/foliate/blob/gtk4/docs/troubleshooting.md#it-cant-open-books-it-hangscrashesshows-a-blank-page

**Comment by jamal-ali1989** [comment]
My bad, I was indeed using Nvidia. If I downgrade to 535 from 560, should that fix the issue?

**Comment by johnfactotum** [comment]
Yes, you could try downgrading as a workaround; or upgrading, if available.

**Comment by ghost** [comment]
I'm having the same issue with Nvidia. Can't open any e-book.

**Comment by jamal-ali1989** [comment]
I have fixed it by adding two environment variables. Use Flatseal and add these: 
LC_ALL=en_US.UTF-8
WEBKIT_DISABLE_DMABUF_RENDERER=1.

**Comment by ghost** [comment]
Doesn't work for me, unfortunately.

**Comment by jamal-ali1989** [comment]
I recommend opening a new issue. Your problem might originate from something else.

**Comment by hepptho** [comment]
I also had crashes trying to open any book or catalog, and `LC_ALL=en_US.UTF-8` fixed it for me.
Thank you for the tip!
